Marlin/stepper.cpp

```diff
@@ -328,13 +329,42 @@ void Stepper::set_directions() {
  *  2000     1 KHz - sleep rate
  *  4000   500  Hz - init rate
  */
-ISR(TIMER1_COMPA_vect) { Stepper::isr(); }
+ISR(TIMER1_COMPA_vect) {
+  #if ENABLED(ADVANCE) || ENABLED(LIN_ADVANCE)
+    Stepper::ISRhandler();
```

Comment:
Not super clean. If the variables needed are out of scope, you can precede them with `stepper.` and if they are `private`, you can make them `protected` instead.

For now let's just leave things as they are.